gub/commands.py

```diff
@@ -341,9 +342,23 @@ def execute (self, logger):
             globs.append ('no-globs-for-%(dest)s' % locals ())
```

Comment:
IIUC it's because of this idiosyncracy that the ``--ignore-failed`` Tar flag is useful, maybe not only for linux-x86:binutils (this was the first package on which I got a consistent build failure without this flag).